worker/backend/backend.go

```diff
@@ -3,14 +3,17 @@ package backend
 import (
 	"context"
 	"fmt"
+	"syscall"
 	"time"
 
 	"code.cloudfoundry.org/garden"
 	"github.com/concourse/concourse/worker/backend/libcontainerd"
 	bespec "github.com/concourse/concourse/worker/backend/spec"
+	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/cio"
+	"github.com/containerd/containerd/errdefs"
 	"github.com/containerd/containerd/namespaces"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-spec/specs-go"
 )
 
 var _ garden.Backend = (*Backend)(nil)
```

Comment:
what does this mean?

Reply:
In the most recent version of go, if a package ends with `-go`, go imports will automatically ignore the suffix and import it as the * in *-go. I think Goland does this as part of running gofmt